persistence/remote/src/main/java/org/infinispan/persistence/remote/logging/Log.java

```diff
@@ -57,4 +57,7 @@ public interface Log extends BasicLogger {
 
    @Message(value = "Segmentation is not supported for a RemoteStore when the configured key media type %s does not match the remote servers key media type %s", id = 10011)
    CacheConfigurationException segmentationRequiresEqualMediaTypes(MediaType cacheMediaType, MediaType serverMediaType);
+
+   @Message(value = "The RemoteCacheStore cannot be segmented when grouping is enabled")
```

Comment:
`id` is not needed?

Reply:
Which `id`? Are you referring to a cache name for example? Unfortunately, our other messages don't have that either.

Reply:
The annotation property, for example in line 58 with `id = 10011`

https://github.com/infinispan/infinispan/blob/db6de87be1c9edbcfb6e354027e39482da6b298d/persistence/remote/src/main/java/org/infinispan/persistence/remote/logging/Log.java#L58

Reply:
Ah, yes. I need the id number 👍